src/dataclay/config.py

```diff
@@ -232,4 +232,8 @@ def set_runtime(new_runtime: Union[ClientRuntime, BackendRuntime]):
     current_runtime = new_runtime
 
 
-logging.basicConfig(level=settings.loglevel)
+def loggerConfig(**kwargs):
+    logging.basicConfig(**kwargs)
+
+
+loggerConfig(level=settings.loglevel)
```

Comment:
This line should not be included, because it is triggering a logging.basicConfig on import time which is what we shuold avoid.